apps/google-analytics-4/frontend/src/components/sidebar/SidebarAnalytics.tsx

```diff
@@ -0,0 +1,46 @@
+import { useAutoResizer, useFieldValue } from '@contentful/react-apps-toolkit';
+import ChartFooter from 'components/sidebar/ChartFooter';
+import ChartHeader from 'components/sidebar/ChartHeader';
+import { useEffect, useState } from 'react';
+import { RunReportResponse } from 'types';
+import { config } from '../../config';
+
+export default function SidebarAnalytics() {
+  const [runReportResponse, setRunReportResponse] = useState<RunReportResponse | undefined>();
+  const [slugValue] = useFieldValue<string | undefined>('slug');
+
+  useAutoResizer();
+
+  useEffect(() => {
+    const baseUrl = config.backendApiUrl;
+
+    async function fetchData() {
+      const response = await fetch(`${baseUrl}/sampleData/runReportResponseHasViews.json`);
+      setRunReportResponse(await response.json());
+    }
+
+    fetchData();
+  }, []);
+
+  const pageViews =
+    runReportResponse &&
+    runReportResponse.rows.reduce((acc, val) => {
+      const metric = Number(val.metricValues[0].value);
```

Comment:
You can use the unary Typescript operator instead of the `Number` class to convert the string value to a number more performantly: 
```
const metric = +val.metricValues[0].value
```